telegram/ext/filters.py

```diff
@@ -320,16 +372,21 @@ def filter(self, message):
             return bool(message.game)
 
     game = _Game()
+    """:obj:`Filter`: Messages that contain :class:`telegram.Game`."""
 
     class entity(BaseFilter):
-        """Filters messages to only allow those which have a :class:`telegram.MessageEntity`
+        """
+        Filters messages to only allow those which have a :class:`telegram.MessageEntity`
         where their `type` matches `entity_type`.
 
+        Note:
+            Example ``MessageHandler(Filters.entity("hashtag"), callback_method)``
```

Comment:
I doubted this. It makes the line too long (which sucks for flake8) and users don;t HAVE to use the constants. The filter works fine with just a string.